### PR TITLE
Adding missing files in aws-cdk preset

### LINKED ISSUE #19: Adding missing files in aws-cdk

### PR DESCRIPTION
**Pull request by ramonmulia**
Closes #19

### CODE REVIEW

File: aws-cdk/index.js [hunk 1]
```diff
@@ -25,8 +25,13 @@ module.exports = function task({ projectName, projectDescription }) {
   deleteFiles('index.js')
   copyFiles(`${__dirname}/static`, [
     '.github/workflows/ci.yml',
+    '.github/workflows/cd.yml',
     '.github/dependabot.yml',
+    './lib/cdk-stack.js',
```
Comment (radomird) [inline aws-cdk/index.js:30]: Shouldn't we copy also the `bin/cdk.js` file  (the one that has been generated above)?

Comment (ramonmulia) [reply aws-cdk/index.js:30]: not in this case, `bin/cdk.js` will be generated by `template` [here](https://github.com/nearform/mrm-preset-nearform/blob/fix/add-missing-files/aws-cdk/index.js#L20) to add the project Id

Comment (radomird) [reply aws-cdk/index.js:30]: ok, understood

**Review by radomird — APPROVED** [review]
lgtm

**Review by katafira — APPROVED** [review]
LGTM.

**Pull request merged**